components/profile/overview/overview.tsx

```diff
@@ -0,0 +1,64 @@
+import { AnimatePresence, motion } from 'framer-motion'
+
+import { Button } from '@/components/common/Button/Button'
+import { Heading } from '@/components/common/Heading'
+import { SectionContainer } from '@/components/common/SectionContainer'
+import Spinner from '@/components/common/Spinner/spinner'
+
+import SeasonPanel from '../career/season-panel'
+import { ExperiencePanel } from './experience/experience-panel'
+import { useProfileOverview } from './hooks'
+import { PlayerBaseData } from './playerBaseData/playerBaseData'
+import { PlayerSlider } from './slider/slider'
+
+export const ProfileOverview = () => {
+  const { data, isLoading } = useProfileOverview('overview')
+
+  return (
+    <AnimatePresence mode="wait">
+      {isLoading ? (
+        <motion.div
+          key="headerLoader"
```

Comment:
again, change to this and remove `Partial<..>` from other components
```suggestion
    <AnimatePresence mode="wait">
      {isLoading || !data ? (
        <motion.div
          key="headerLoader"
```